### DISCUSSION TITLE
GitHub markdown/HTML: lazy loading images

### DISCUSSION THREAD

**Discussion by fballiano** [community/community#42558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When you insert an image in a textfield (or in a README file), sometime you'd like to have lazy loading for images (ex: the contributors list in a project's README can have megabytes of images downloaded at every request) but if you write something like this:

```<img src="https://user-images.githubusercontent.com/909743/209436052-7e97fecf-2cc8-4795-bbc6-eb227347273c.jpg" loading=lazy />```

the output will be sanitized and the `loading` attribute will be removed, eg:

<img src="https://user-images.githubusercontent.com/909743/209436052-7e97fecf-2cc8-4795-bbc6-eb227347273c.jpg" loading=lazy />

If would be nice (actually it would be quite necessary in 2023) to have the loading attribute not stripped away.

:-)

**Comment by johnnyreilly** [comment]
It would be amazing to have this! Posting here after investigating https://github.com/JoshuaKGoldberg/create-typescript-app/issues/1312
